src/services/dispatchers.ts

```diff
@@ -67,6 +65,31 @@ async function createDispatcher(sdk: OAuth1Client, dispatcher: DispatcherCreatio
   console.log(green(`Created Dispatcher: ${dispatcher.name} ✓`));
 }
 
+async function updateDispatcher(sdk: OAuth1Client, localDispatcher: DispatcherCreation, exhDispatcher: Dispatcher) {
+  console.group(blue(`Updating Dispatcher: ${localDispatcher.name}`));
+  await dispatcherRepository.update(sdk, exhDispatcher.id, localDispatcher);
+
+  // Updating a Dispatcher will not update the actions, thus they must be synchronized separately
+  await synchronizeActions(sdk, localDispatcher, exhDispatcher);
+
+  console.groupEnd();
+  console.log(green(`Updated Dispatcher: ${localDispatcher.name} ✓`));
```

Comment:
For the validation I put the check in front because it looks cleaner imo. Should we do this everywhere? Maybe in a small follow up PR to clean up the output like we did yesterday.

Reply:
So I put it after because it's like that in all of the Schema verification, but we'll change that next cycle anyways, so go for it.